assessment.rb

```diff
@@ -1,12 +1,43 @@
 require 'csv'
 # Assessment answers to:
-
+rails g model Licensee surname given_name gender height:integer dob:date hair_color eye_color
 # question 2:
+Licensee.create
+surname: 'Bushop',
+given_name: 'Carrie'
+gender: 'f',
+height: 63,
+dob: '1943-09-26'
+hair_color: 'Brown'
+eye_color: 'Black'
```

Comment:
You should indent these key-value pairs - as written, it's a little difficult to parse.